nozzle/config.go

```diff
@@ -0,0 +1,134 @@
+package nozzle
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/cloudfoundry/sonde-go/events"
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config holds users provided env variables
+type Config struct {
+	Nozzel    *NozzelConfig
+	WaveFront *WaveFrontConfig
+}
+
+// NozzelConfig holds specific PCF env variables
+type NozzelConfig struct {
+	APIURL                 string `required:"true" envconfig:"api_url"`
+	Username               string `required:"true"`
```

Comment:
Is there a reason username and pwd don't have an `envconfig`?

Reply:
Because the struct field name match the env var.